### PR TITLE
React css collapse

### PR DESCRIPTION
**Pull request by dford07**
Please fill in this template.

- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`

### CODE REVIEW

File: types/react-css-collapse/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// Type definitions for react-css-collapse 3.6.0
+// Project: https://github.com/SparebankenVest/react-css-collapse;
```
Comment (amcasey) [inline types/react-css-collapse/index.d.ts]: Stray semicolon?

File: types/react-css-collapse/tslint.json [hunk 2]
```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "adjacent-overload-signatures": false,
```
Comment (amcasey) [inline types/react-css-collapse/tslint.json]: Why disable these lint rules?

File: types/react-css-collapse/tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": false,
```
Comment (amcasey) [inline types/react-css-collapse/tsconfig.json]: Why disable null checks?

**Review by amcasey — CHANGES_REQUESTED** [review]

**Comment by dford07** [comment]
hi @amcasey i have revised your suggestions and made my amendments

File: types/react-css-collapse/index.d.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
+// Type definitions for react-css-collapse 3.6
+// Project: https://github.com/SparebankenVest/react-css-collapse
+// Definitions by: Daniel Ford <https://github.com/dford07>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+declare module 'react-css-collapse' {
```
Comment (amcasey) [inline types/react-css-collapse/index.d.ts]: I think this module declaration is redundant - the exports already come from `react-css-collapse`.

**Review by amcasey — APPROVED** [review]
Thanks!  One more minor comment.

File: types/react-css-collapse/index.d.ts (hunk 4 above)
Comment (dford07) [reply types/react-css-collapse/index.d.ts]: is it worth me changing this?

**Pull request merged**